x-pack/metricbeat/module/aws/ec2/data.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package ec2
+
+import (
+	"github.com/elastic/beats/libbeat/common/schema"
+	"github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	s = schema.Schema{
```

Comment:
I think you are not using this? Normally we use this to parse structured data (ie a JSON response) into desired field names